config/i18n-tasks.yml

```diff
@@ -94,6 +94,7 @@ ignore_unused:
 - 'errors.not_authorized'
 - 'experiments.*'
 - 'event_types.*'
+- 'headings.create_account_with*' # cannot find with interpolation
```

Comment:
I agree with Jessie. Conditionals in views should be avoided. This situation calls for Replacing Conditional with Polymorphism:
https://robots.thoughtbot.com/from-model-logic-to-views-helpers-and-to-null-objects
https://sourcemaking.com/refactoring/replace-conditional-with-polymorphism